.github/opensearch-cluster/docker-compose.yml

```diff
@@ -4,8 +4,8 @@ services:
   opensearch-cluster:
     image: opensearchproject/opensearch:${OPENSEARCH_VERSION:-latest}
     ports:
-      - "9200:9200"
-      - "9600:9600"
+      - '9200:9200'
+      - '9600:9600'
     environment:
-      - "discovery.type=single-node"
-      - "OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_PASSWORD}"
\ No newline at end of file
+      - discovery.type=single-node
+      - 'OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_PASSWORD}'
```

Comment:
Personally I prefer being able to add the quotes on line even though it's not necessary for that specific case. It just makes it easier to edit and you don't have to keep removing and adding the quotes as the value changes. It's the same reason some languages, like JS itself, allows trailing comma like the comma after "world" in:
```javascript
{ 
   foo: "bar",
   hello: "world",
}
``` 
so that when you add or a remove a property after `hello: "world"` you dont have to keep removing or adding the comma. There's no harm in keeping the quotes either. So I think this lint is not necessary.

Reply:
The lint makes it possible to not have to think about it, and not get inconsistent format for VARIABLES, where some are one way and others are the other way.

I’ll remove though, but lmk if this changes your mind.